grade_test.go

```diff
@@ -261,14 +269,33 @@ func TestRunCompiled(t *testing.T) {
 		t.Errorf("Failed to open the compiled file: [err=%e]", err)
 	}
 
-	expected := "Hello world!\n"
-	actual := runCompiled(dir, "", []string{})
+	expected := "Hello, C Plus Plus!\n"
+	actual := runCompiled(dir, tmp.Name(), "c++", []string{"Hello, C Plus Plus\n"})
```

Comment:
arg pass doesn't make sense

Reply:
I think I misunderstood the last argument's use case. Going to fix.